src/clib/pioc_support.c

```diff
@@ -355,7 +444,7 @@ int check_mpi(file_desc_t *file, int mpierr, const char *filename,
  * @param line the line of code where error occured.
  * @return PIO_NOERR for no error, otherwise PIO_EIO.
  */
-int check_mpi2(iosystem_desc_t *ios, file_desc_t *file, int mpierr,
+int check_mpi(iosystem_desc_t *ios, file_desc_t *file, int mpierr,
```

Comment:
I would like to learn more about the motivation for the changes in error handling.